ci-operator/step-registry/storage/conf/csi-openstack-manila/storage-conf-csi-openstack-manila-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: storage-conf-csi-openstack-manila
+  # Canonical location of the CSI driver manifest
+  from: manila-csi-driver-operator-test
```

Comment:
Does this image exist?
Tests fail with:
```
ImagePullBackOff and message Back-off pulling image "image-registry.openshift-image-registry.svc:5000/ci-op-15kmb88s/stable:manila-csi-driver-operator-test
```

Reply:
To create the image add 

```yaml
- dockerfile_path: Dockerfile.test
  from: src
  to: manila-csi-driver-operator-test
```

[here](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/csi-driver-manila-operator/openshift-csi-driver-manila-operator-master.yaml#L20)

and run `make jobs` again.